2024-02-05 18:37:15,371 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:37:18,667 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:37:18,699 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:37:18,699 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:37:43,251 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:37:43,268 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:37:43,268 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:43:29,128 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:43:29,146 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:43:29,146 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:43:31,188 ERROR: Exception on /products/lista [GET] [in /home/axel/.local/lib/python3.10/site-packages/flask/app.py:825]
Traceback (most recent call last):
  File "/home/axel/.local/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/axel/.local/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/axel/.local/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/axel/.local/lib/python3.10/site-packages/flask_debugtoolbar/__init__.py", line 157, in dispatch_request
    return view_func(**req.view_args)
  File "/home/axel/Escritorio/m012/proyectoM12/wannapop/routes_main.py", line 26, in product_list
    return render_template('products/list.html', products_with_category=products_with_category)
  File "/home/axel/.local/lib/python3.10/site-packages/flask/templating.py", line 152, in render_template
    return _render(app, template, context)
  File "/home/axel/.local/lib/python3.10/site-packages/flask/templating.py", line 133, in _render
    rv = template.render(context)
  File "/home/axel/.local/lib/python3.10/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/home/axel/.local/lib/python3.10/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/axel/Escritorio/m012/proyectoM12/wannapop/templates/products/list.html", line 1, in top-level template code
    {% extends '_template.html' %}
  File "/home/axel/Escritorio/m012/proyectoM12/wannapop/templates/_template.html", line 34, in top-level template code
    {% block content %} {% endblock %}
  File "/home/axel/Escritorio/m012/proyectoM12/wannapop/templates/products/list.html", line 18, in block 'content'
    {% set banned = banned_dict.get(product.id) %}
  File "/home/axel/.local/lib/python3.10/site-packages/jinja2/environment.py", line 485, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'banned_dict' is undefined
2024-02-05 18:43:31,188 ERROR: Exception on /products/lista [GET] [in /home/axel/.local/lib/python3.10/site-packages/flask/app.py:825]
Traceback (most recent call last):
  File "/home/axel/.local/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/axel/.local/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/axel/.local/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/axel/.local/lib/python3.10/site-packages/flask_debugtoolbar/__init__.py", line 157, in dispatch_request
    return view_func(**req.view_args)
  File "/home/axel/Escritorio/m012/proyectoM12/wannapop/routes_main.py", line 26, in product_list
    return render_template('products/list.html', products_with_category=products_with_category)
  File "/home/axel/.local/lib/python3.10/site-packages/flask/templating.py", line 152, in render_template
    return _render(app, template, context)
  File "/home/axel/.local/lib/python3.10/site-packages/flask/templating.py", line 133, in _render
    rv = template.render(context)
  File "/home/axel/.local/lib/python3.10/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/home/axel/.local/lib/python3.10/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/axel/Escritorio/m012/proyectoM12/wannapop/templates/products/list.html", line 1, in top-level template code
    {% extends '_template.html' %}
  File "/home/axel/Escritorio/m012/proyectoM12/wannapop/templates/_template.html", line 34, in top-level template code
    {% block content %} {% endblock %}
  File "/home/axel/Escritorio/m012/proyectoM12/wannapop/templates/products/list.html", line 18, in block 'content'
    {% set banned = banned_dict.get(product.id) %}
  File "/home/axel/.local/lib/python3.10/site-packages/jinja2/environment.py", line 485, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'banned_dict' is undefined
2024-02-05 18:43:41,230 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:43:41,246 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:43:41,246 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:43:41,760 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:43:41,777 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:43:41,777 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:44:22,181 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:44:22,198 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:44:22,198 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:44:25,895 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:44:25,912 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:44:25,912 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:47:01,700 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:47:01,717 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:47:01,717 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:47:11,589 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:47:11,607 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:47:11,607 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:47:16,507 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:47:16,526 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:47:16,526 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:03,895 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:03,912 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:03,912 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:05,681 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:05,699 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:05,699 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:48,147 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:48,164 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:48,164 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:56,999 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:57,017 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:51:57,017 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:52:01,269 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:52:01,286 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:52:01,286 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:52:01,850 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:52:01,866 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
2024-02-05 18:52:01,866 INFO: Aplicación iniciada [in /home/axel/Escritorio/m012/proyectoM12/wannapop/__init__.py:61]
